src/data-workspace/data-details-sidebar/comment.test.js

```diff
@@ -145,7 +145,7 @@ describe('<Comment />', () => {
             return {
                 mutate: () => {
                     // fire after returning
-                    setTimeout(onSuccess, 0)
+                    setTimeout(() => onSuccess({}), 0)
```

Comment:
This test got broken because I am now passing the saved data value object to the `onSucces` handler, so had to apply this fix.